app/ui/preload/index.js

```diff
@@ -15,6 +15,10 @@ import './active-element';
 import Readability from './readability';
 import { readerify } from './reader';
 import { parseMetadata } from './metadata-parsing';
+import { reload } from './reload';
 
 window._readerify = readerify.bind(null, Readability);
 window._parseMetadata = parseMetadata;
+// Expose _TOFINO_RELOAD for the tofino://sessionrestore URLs that need
+// the webview to reload itself
+window._TOFINO_RELOAD = reload;
```

Comment:
Although we could conditionally attach it to window only on `tofino://sessionrestore` pages